packages/fx-core/resource/package.nls.json

```diff
@@ -323,12 +323,9 @@
   "core.createProjectQuestion.option.description.worksInOutlook": "Works in Teams and Outlook",
   "core.createProjectQuestion.option.description.worksInOutlookM365": "Works in Teams, Outlook, and the Microsoft 365 application",
   "core.createProjectQuestion.projectType.bot.detail": "Create instant, engaging chat experiences that automate tasks seamlessly",
-  "core.createProjectQuestion.projectType.agentForTeams.label": "Agent for Teams",
-  "core.createProjectQuestion.projectType.agentForTeams.detail": "Build an agent for Microsoft Teams easily using Teams AI Library",
-  "core.createProjectQuestion.projectType.agentForTeams.title": "Agent for Teams Using Teams AI Library",
-  "core.createProjectQuestion.projectType.teamsApp.label": "Teams App",
-  "core.createProjectQuestion.projectType.teamsApp.detail": "Create your own web page, chat bot or message extension for Microsoft Teams",
-  "core.createProjectQuestion.projectType.teamsApp.title": "App Features Used in Microsoft Teams",
+  "core.createProjectQuestion.projectType.teamsAgentsAndApps.label": "Teams Agents and Apps",
+  "core.createProjectQuestion.projectType.teamsAgentsAndApps.detail": "Create your own agent, web page, chat bot or message extension for Microsoft Teams easily using Teams AI Library",
```

Comment:
The sentence is too long to show in the UI, can shorten to:
"Create agent, chat bot, web pages or other apps in Microsoft Teams using Teams AI library."